craft_application/launchpad/models/recipe.py

```diff
@@ -370,13 +386,27 @@ def new(  # noqa: PLR0913
         )
         cls._fill_repo_info(kwargs, git_ref=git_ref)
 
-        charm_entry = lp.lp.charm_recipes.new(
-            name=name,
-            owner=util.get_person_link(owner),
-            project=f"/{project}",
-            auto_build=auto_build,
-            **kwargs,
-        )
+        charm_entry = None
+
+        for attempt in range(_ATTEMPT_COUNT):
+            try:
+                charm_entry = lp.lp.charm_recipes.new(
```

Comment:
emit a debug message here, something like f"Trying to Create project ({attempt}/{_ATTEMPT_COUNT}"